types/plan/plan.go

```diff
@@ -38,11 +38,12 @@ type Spec struct {
 	SourceDir         string `yaml:"sourceDir"`
 	CustomizationsDir string `yaml:"customizationsDir,omitempty"`
 
-	Services                     map[string][]PlanArtifact `yaml:"services"` //[servicename]
-	InvokedByDefaultTransformers []string                  `yaml:"invokedByDefaultTransformers,omitempty"`
+	Services map[string][]PlanArtifact `yaml:"services"` //[servicename]
 
-	TransformerSelector metav1.LabelSelector `yaml:"transformerSelector,omitempty"`
-	Transformers        map[string]string    `yaml:"transformers,omitempty" m2kpath:"normal"` //[name]filepath
+	TransformerSelector          metav1.LabelSelector `yaml:"transformerSelector,omitempty"`
+	Transformers                 map[string]string    `yaml:"transformers,omitempty" m2kpath:"normal"` //[name]filepath
+	InvokedByDefaultTransformers []string             `yaml:"invokedByDefaultTransformers,omitempty"`
+	DisabledTransformers         []string             `yaml:"disabledTransformers,omitempty"`
```

Comment:
It might be good to have the path here too. Probably as a 2d array to allow for more than one path for a transformer.